makefile

```diff
@@ -1,40 +1,60 @@
-start: erase build db up
+.PHONY: start
+start: erase build db up ## clean current environment, recreate dependencies and spin up again
 
-stop:
+.PHONY: stop
+stop: ## stop environment
 		docker-compose stop
 
-rebuild: start
+.PHONY: rebuild
+rebuild: start ## same as start
 
-erase:
+.PHONY: erase
+erase: ## stop and delete containers, clean volumes.
 		docker-compose stop
 		docker-compose rm -v -f
 
-build:
+.PHONY: build
+build: ## build environment and initialize composer and project dependencies
 		docker-compose build
-		docker-compose run php sh -lc 'composer install'
+		docker-compose up -d
+		docker-compose exec php sh -lc 'composer install'
```

Comment:
Composer install can be stateless

Reply:
`docker-compose run --rm php sh -lc 'composer install'` its enough here and there's no need to up all the enviroment